spec/models/events/pull_request_spec.rb

```diff
@@ -62,4 +62,24 @@
       expect(subject).to_not be_valid
     end
   end
+
+  describe 'callbacks' do
+    context 'when a pull request is merged' do
+      before { travel_to(Time.zone.today.beginning_of_day) }
+      let!(:pull_request) { create(:pull_request, opened_at: Time.current) }
+
+      it 'creates a merge time with the correct values' do
+        pull_request.update!(merged_at: Time.current)
+        merge_time = MergeTime.last
+        expect(merge_time[:value]).to eq(0)
```

Comment:
Using `0` is a bit confusing to me. Perhaps you can do:

`pull_request.update!(merged_at: Time.current + 1.hour)`

and then

`expect(merge_time[:value]).to eq(3600)`

WDYT?